gravityflow/gravityflowwoocommerce
# Fix variable products was added to cart twice

re:[HS#12709](https://secure.helpscout.net/conversation/1082902961/12709/)

This PR resolves an issue that variable products are added to cart twice with our Checkout step.

The issue was about we had a conflict on handling our own `gflow_access_token` GET param and WooCommerce's `add-to-cart` GET param. The revised code basically reflects that.

## Testing instructions
1. Read the original ticket for more details. 
2. If you don't have WooCommerce Subscriptions, you can still reproduce it with a variable WooCommerce product. See how to use `add-to-cart` param in the Checkout Step here: https://docs.gravityflow.io/article/247-woocommerce-extension-checkout-step
3. In the master branch you'll get the product added to cart twice, which will get you 2 items of the same product (reflect on quantity). Switch to this branch should resolve it.
